server/cmd/mmctl/commands/channel_e2e_test.go

```diff
@@ -354,7 +354,7 @@ func (s *MmctlE2ETestSuite) TestDeleteChannelsCmd() {
 		_, err = s.th.App.GetChannel(s.th.Context, channel.Id)
 
 		s.Require().NotNil(err)
-		s.Require().Equal(fmt.Sprintf("GetChannel: Unable to find the existing channel., resource: Channel id: %s", channel.Id), err.Error())
+		s.Require().Equal(fmt.Sprintf("GetChannel: Unable to find the existing channel., resource \"Channel\" not found, id: %s", channel.Id), err.Error())
```

Comment:
How about this? We already say what the resource/entity is. In fact, we can discard the "entity" bit as well.
```suggestion
		s.Require().Equal(fmt.Sprintf("GetChannel: Unable to find the existing channel., entity id: %s", channel.Id), err.Error())
```

Reply:
I like having the `entity` bit as some parts of the codebase don't do error wrapping. This can make it hard to understand what entity was not found. Also, it would require a large diff to remove the `entity` argument.